openedx_lti_tool_plugin/tests/views/test_launch_view.py

```diff
@@ -775,8 +776,8 @@ def test_get_unit_component_launch_response_with_wrong_block_type(
 class TestLtiToolLaunchViewHandleAgs(TestLtiToolLaunchViewBase):
     """Testcase for LtiToolLaunchView handle_ags method."""
 
-    def test_handle_ags(self, lti_graded_resource_mock: MagicMock):
-        """Test the 'handle_ags' method when the launch message contains AGS data.
+    def test_handle_ags_with_existent_resource(self, lti_graded_resource_mock: MagicMock):
+        """Test the 'handle_ags' method when the resource doesn't exists.
```

Comment:
```suggestion
    def test_handle_ags_with_existent_resource(self, lti_graded_resource_mock: MagicMock):
        """Test the 'handle_ags' method when the resource doesn't exists.
```
Why is the name of the test different from the description?, I think we should keep it just has test_handle_ags and "Test the 'handle_ags' method" since this is the happy path.